colors.py

```diff
@@ -49,3 +49,37 @@ def nearest_match(self):
                     min = abs(item[1] - color_int)
                     color = item[0]
             return "Nearest Match: " + color
+    def RGB_cartesian_match(self):
+        #eucledian
+        color = None
+        min = 16777215  # FFFFFF
+        for item in colors[self.colset].values():
+            colorCompareHex = "0x{:06x}".format(item[1])
```

Comment:
Variable names should be lowercase, with words separated by underscores as necessary to improve readability.

What are we doing here?
I think we are converting the integer value to hex. If that's the case then we can drop this integer part of the item because we can obtain it directly from the key.
Will make that change in a different commit.
This is good for now.